cmd/convox/rack.go

```diff
@@ -215,7 +221,19 @@ func cmdRackParamsSet(c *cli.Context) error {
 		return stdcli.Error(err)
 	}
 
-	fmt.Println("OK")
+	if c.Bool("wait") {
+		stdcli.Startf("Waiting for completion")
+
+		// give the rack a few seconds to start updating
+		time.Sleep(5 * time.Second)
+
+		if err := waitForRackRunning(c); err != nil {
+			return stdcli.Error(err)
+		}
+
+		stdcli.OK()
```

Comment:
Sorry, I don't see what you mean here. I was trying to use the same style as [here](https://github.com/convox/rack/blob/master/cmd/convox/rack.go#L304-L315), is that not the way to go?